doc-content/enterprise-only/release-notes/rn-tech-preview-ref.adoc

```diff
@@ -10,15 +10,29 @@ These features are for Technology Preview only. Technology Preview features are
 For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
 ====
 
-== High-availability authoring on {EAP} (on premise and with OpenShift) 
+== High-availability authoring on {EAP} (on premise and with OpenShift)
 Configuration of {CENTRAL} for high availability is currently technology preview. Search and messaging technology used to enable search feature is considered an implementation detail. {PRODUCT} may choose to replace that technology in the future while preserving the capability offered.
- 
+
 ifdef::PAM[]
 == Case modeler
-Case modeler now provides the option to model a case as a sequence of stages so it is simple to define a case model at high-level. Case modeling supports three types of tasks: human tasks, sub-processes, and sub-cases. 
+Case modeler now provides the option to model a case as a sequence of stages so it is simple to define a case model at high-level. Case modeling supports three types of tasks: human tasks, sub-processes, and sub-cases.
 
 [NOTE]
 ====
 The case modeler in {PRODUCT} {PRODUCT_VERSION} is a Technology Preview feature and is disabled by default in {CENTRAL}. To enable the case modeler preview in {CENTRAL}, in the upper-right corner of the window click *Settings* -> *Roles*, select a role from the left panel, click *Editors* -> *(New) Case Modeler* -> *Read*, and then click *Save* to save the changes.
 ====
 endif::PAM[]
+
+== Operator installer wizard
+An installer wizard is provided in the OpenShift Operator for {PRODUCT}. You can use the wizard to deploy a {PRODUCT} environment on {OPENSHIFT} with the Operator.
+
+== Process Instance Migration as a standalone service
+Process instance migration is now available as a standalone service containing a user interface and a backend, and packaged as a Thorntail uber-jar. The process instance migration service enables a user to define the migration between two different process definitions, known as a migration plan. The user can apply the migration plan to the running process instances in a specific {KIE_SERVER}.
```

Comment:
Curious about the Thorntail reference. It is factually correct, but isn't there some kind of a productized name for it? Thorntail is an upstream project name.

Reply:
I checked and there isn't a productized name that I was able to find.